doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1050,6 +1050,25 @@ Before being moved there,
   on your machine, large file uploads via API will cause RAM and/or swap usage bursts. You might want to point this to
   a different location, restrict maximum size of it, and monitor for stale uploads.
 
+.. _cache-rate-limiting:
+
+Configure Your Dataverse Installation to use JCache (with Hazelcast as a provider) for Rate Limiting
```

Comment:
Technically, we're not using JCache here, just an IMDG provided by Payara using Hazelcast.

Reply:
Switched to use javax.cache.Cache class